docs/user/environment-variables.md

```diff
@@ -27,7 +27,7 @@ The installer accepts a number of environment variable that allow the interactiv
      The container registry pull secret for this cluster (e.g. `{"auths": {...}}`).
      You can get this secret from [try.openshift.com](https://try.openshift.com).
 * `OPENSHIFT_INSTALL_PULL_SECRET_PATH`:
-     As an alternative to `OPENSHIFT_INSTALL_SSH_PUB_KEY`, you can configure this variable with a path containing your pull secret.
+     As an alternative to `OPENSHIFT_INSTALL_PULL_SECRET`, you can configure this variable with a path containing your pull secret.
```

Comment:
Can we get this in a separate PR.  I know we want this, but I need to think more about the `build.sh` change

Reply:
sure: https://github.com/openshift/installer/pull/850